### DISCUSSION TITLE
Test and Service Button not working

### DISCUSSION THREAD

**Discussion by owenthesaint** [flyinghead/flycast#1055] [Q&A]
Hi there,
I'm having a strange issue with Flycast,
I tried to map test and service buttons to many different keyboards or controllers,
but naomi just does not seem to register that i'm pushing those buttons.
I have tried with many games but still nothing happens.
would there be something i might of missed in config?

**Comment by flyinghead** [comment]
Are you using RetroArch? If that's the case, there's a core option to enable test and service buttons.

There's no such option on standalone so Test and Service buttons are always active.

**Reply by owenthesaint** [reply]
I'm using the windows standalone version,
I will re-download flycast and try with a fresh setup to see if anything changes.
Thanks for getting back to me.
I will let you know how it goes.

**Reply by owenthesaint** [reply]
Ok so I found what was causing this, I seemed to have bad BIOS files.
Thanks again !!!
